spark/src/main/scala/org/apache/comet/serde/QueryPlanSerde.scala

```diff
@@ -2889,6 +2857,48 @@ object QueryPlanSerde extends Logging with CometExprShim {
     }
   }
 
+  /**
+   * Determine which data types are supported as hash-partition keys in a shuffle.
+   */
+  def supportedShufflePartitionDataType(dt: DataType): Boolean = dt match {
```

Comment:
```suggestion
  def supportedShufflePartitionKeyDataType(dt: DataType): Boolean = dt match {
```

Reply:
I applied this change myself since I had to update the caller sites as well.